tox.ini

```diff
@@ -81,7 +81,9 @@ deps =
     -r{toxinidir}/requirements.txt
     pytest
     cosl
-    ops-scenario > 4
+    # fixme replace with scenario>-6.1.5
+    git+http://github.com/canonical/ops-scenario@fix-juju-info-network-get
```

Comment:
Reminder to clear this up.

Reply:
that can be now cleaned up. 6.1.5 is released